src/store/ui-slice.ts

```diff
@@ -1,27 +1,85 @@
 import { createSlice, PayloadAction } from '@reduxjs/toolkit'
 import type { SupportedVisType } from '@constants/visualization-types'
+import { getUserSidebarWidthFromLocalStorage } from '@modules/local-storage'
 
 export interface UiState {
     visualizationType: SupportedVisType
+    hideLayoutPanel: boolean
+    hideMainSidebar: boolean
+    accessoryPanelWidth: number
+    showAccessoryPanel: boolean
+    showDetailsPanel: boolean
+    showExpandedLayoutPanel: boolean
 }
 
 export const initialState: UiState = {
     visualizationType: 'LINE_LIST',
+
+    hideLayoutPanel: false,
```

Comment:
These are copied from LL.
I wonder about the naming, there's a mix of `hide` and `show` here, I assume the reason is the default state, if the panel is showing by default, the key/action is `hide` and viceversa.